arrayMethods2.js

```diff
@@ -0,0 +1,107 @@
+//QUESTION 1
```

Comment:
Good job on committing often. But remember to start the commit comment with a verb. "add question 3", "fix syntax"